### PR TITLE
[IdentityLookup] Updates for Xcode14 beta1

### PR DESCRIPTION
**Pull request by tj-devel709**

### CODE REVIEW

File: src/identitylookup.cs [hunk 1]
```diff
@@ -100,6 +119,10 @@ interface ILMessageFilterQueryRequest : NSSecureCoding {
 
 		[NullAllowed, Export ("messageBody")]
 		string MessageBody { get; }
+
+		[NoWatch, NoTV, NoMac, MacCatalyst (16,0), iOS (16,0)]
+		[NullAllowed, Export ("receiverISOCountryCode")]
+		string ReceiverISOCountryCode { get; }
```
Comment (dalexsoto) [inline src/identitylookup.cs:125]:
```suggestion
		string ReceiverIsoCountryCode { get; }
```

File: src/identitylookup.cs [hunk 2]
```diff
@@ -215,4 +242,31 @@ interface ILMessageCommunication {
 		[Export ("isEqualToMessageCommunication:")]
 		bool IsEqualTo (ILMessageCommunication communication);
 	}
+
+	[NoWatch, NoTV, NoMac, MacCatalyst (16,0), iOS (16,0)]
+	[Protocol]
+	interface ILMessageFilterCapabilitiesQueryHandling
+	{
+		[Async]
```
Comment (dalexsoto) [inline src/identitylookup.cs:250]:
```suggestion
```
No Async on protocols

File: src/identitylookup.cs [hunk 3]
```diff
@@ -215,4 +242,31 @@ interface ILMessageCommunication {
...
+
+	[NoWatch, NoTV, NoMac, MacCatalyst (16,0), iOS (16,0)]
+	[BaseType (typeof (NSObject))]
+	interface ILMessageFilterCapabilitiesQueryResponse : NSSecureCoding
+	{
+		[Export ("transactionalSubActions", ArgumentSemantic.Copy)]
+		NSNumber[] TransactionalSubActions { get; set; }
+
+		[Export ("promotionalSubActions", ArgumentSemantic.Copy)]
+		NSNumber[] PromotionalSubActions { get; set; }
```
Comment (dalexsoto) [inline src/identitylookup.cs:264]: Both of these properties according to [documentation](https://developer.apple.com/documentation/sms_and_call_reporting/ilmessagefiltercapabilitiesqueryresponse) are `ILMessageFilterSubAction []`

```suggestion
		[BindAs (typeof (ILMessageFilterSubAction []))]
		[Export ("transactionalSubActions", ArgumentSemantic.Copy)]
		NSNumber[] TransactionalSubActions { get; set; }

		[BindAs (typeof (ILMessageFilterSubAction []))]
		[Export ("promotionalSubActions", ArgumentSemantic.Copy)]
		NSNumber[] PromotionalSubActions { get; set; }
```

**Review by dalexsoto — CHANGES_REQUESTED** [review]

**Review by rolfbjarne — APPROVED** [review]

File: src/identitylookup.cs (hunk 3 above)
Comment (tj-devel709) [reply src/identitylookup.cs:264]: Ah good catch!

**Review by SotoiGhost — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**